### PR TITLE
Fix python packaging

### PR DESCRIPTION
**Pull request by bzz**
This PR introduce couple of changes

 - [x] move `*.proto` to `lookout/sdk/*.proto`
 - [x] fixes packaging/deployment of python library, under the nice namespace of `lookout.sdk`
 - [x] get's rid of re-writing .proto files, that is [required otherwise](https://github.com/src-d/style-analyzer/blob/99e8a47d97f447ed0bc25e195f37e913c1db39ea/lookout/core/generate_rpc.sh#L12)
 - [x] ~that, by necessity, also changes golang SDK path. Now it should be imported as `gopkg.in/src-d/lookout-sdk.v0/golang/lookout/sdk`~
 - [x] Golang generated code does not depend on package name, so can be safley moved to `gopkg.in/src-d/lookout-sdk.v0/pb` 🎉 

~It makes Golang imports a bit longer, but allows for nice namespaces on python side and does not involve error-prone pre-processing of .proto files any more, so benefits overweight downsides.~

After this is merged, I will cut a release and make a subsequent changes to `sonarcheck-analyzer` ~of imports to lookout, gometalint-analyzer.~ no changes are necessary on Golang side.

### CODE REVIEW

**Comment by bzz** [comment]
@smacker observation that generated Go code does not import from it's package allows us to tweak final package name by just moving code back to `./pb`

File: _tools/protogen_golang.sh [hunk 1]
```diff
@@ -4,14 +4,37 @@
...
+    echo "Failed to run protoc on ${src}/${sdk}"
+    exit 2
+fi
+
+if ! mv "${dst}/${sdk}/"*.go pb ; then
+    echo "Failed to mv ${dst}/${sdk}/*.go to ./pb"
+    exit 2
+fi
+
+if ! rm -rf "${dst}/${sdk}/" ; then
```
Comment (smacker) [inline _tools/protogen_golang.sh]: `rm -rf "${dst}/"`?

File: _tools/protogen_golang.sh [hunk 2]
```diff
@@ -4,14 +4,37 @@
...
+    echo "Failed to delete ${dst}/${sdk}"
+    exit 2
+fi
```
Comment (smacker) [inline _tools/protogen_golang.sh]: new line

**Comment by bzz** [comment]
Thank you for prompt feedback @smaker, addressed in 1dee8ad

**Comment by bzz** [comment]
CI is reeealy snow, it's green under https://travis-ci.org/bzz/lookout-sdk/builds/425273071

**Pull request merged**